src/server/index.js

```diff
@@ -82,6 +83,22 @@ router.get('/boards', function(req, res) {
   }
 });
 
+router.get('/boards/:id', jsonParser, function(req, res) {
```

Comment:
don't need to add `jsonParser` as a middleware here, it's already added to every endpoint